app/controllers/redirect/partner_exit_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Redirect
+  class PartnerExitController < Redirect::ReturnToSpController
```

Comment:
I don't typically see us using the terms "partner" or "exit" in code when referencing the integration that a user will be returned to. Previously we discussed using "cancel" instead. I also think that since this pathway comes directly from the "completions" routes, we should reuse that terminology.

Separately, everything inside `app/controllers/redirect` today is exclusively concerned with HTTP redirects, and there are no user-facing views for any of the existing controllers. I might suggest we move this controller out of that folder / module if we want to maintain that convention.

What do you think about calling this `CompletionsCancellationController` instead, at `app/controllers/completions_cancellation_controller.rb`? (And maybe updating events accordingly)